release-blocking-jobs.md

```diff
@@ -47,6 +47,7 @@ meet the below criteria completely due to technical debt.  Blocking jobs must:
 - Run at least every 3 hours
 - Be able to pass 3 times in a row against the same commit
 - Be Owned by a SIG, or other team, that is responsive to addressing failures, and whose alert email is configured in the job.
+- Depend on accounts resources owned by a SIG through SIG K8s Infra, so the project has visibility into funding and management of resources
```

Comment:
Potential improvement/typo:
```suggestion
- Depend on accounts and resources owned by a SIG through SIG K8s Infra, so the project has visibility into funding and management of resources
```

Reply:
good catch, I also clarified "owned by a SIG through SIG K8s Infra" to "owned by The Kubernetes Project though SIG K8s Infra"